plugins/kiali-backend/package.json

```diff
@@ -45,19 +41,16 @@
   },
   "configSchema": "config.d.ts",
   "dependencies": {
-    "@backstage/backend-common": "^0.23.3",
+    "@backstage/backend-common": "^0.25.0",
```

Comment:
```suggestion
```
`@backstage/backend-common` is deprecated and needs to be removed where possible.

Reply:
Let's stick with your approach for now until https://github.com/backstage/backstage/issues/26933 is resolved.